test/e2e/elasticache/replication_group/coverage_tests/rg_scaling.sh

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env bash
```

Comment:
maybe call this file `test/e2e/elasticache/replication_group/update.sh`

Reply:
I think I'll prefix the files as "update_scaling" and "update_replication" to highlight that they're both update tests, but I think we should keep them separate for now. The test organization is definitely going to change because we plan to add more, and we plan to chain many update operations together in a single test (rather than creating a replication group for each case and performing 1-2 operations on it, which takes a very long time).